src/main/java/de/tum/in/www1/artemis/service/dto/CoverageReportAndSubmissionDateDTO.java

```diff
@@ -0,0 +1,11 @@
+package de.tum.in.www1.artemis.service.dto;
+
+import java.time.ZonedDateTime;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.hestia.CoverageReport;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public record CoverageReportAndSubmissionDateDTO(CoverageReport coverageReport, ZonedDateTime submissionDate) {
```

Comment:
Something I just realise now, I kinda dislike that it's named "DTO". Because it actually functions as a DAO (Data Access Object).
Also, if you don't send it to the client, you don't need the annotation.

Reply:
As discussed, this will be fixed in a follow-up together with other occurrences to form a uniform approach.